src/backend/api/Fusion.Resources.Api/Controllers/Requests/InternalRequestsController.cs

```diff
@@ -187,6 +186,10 @@ public class InternalRequestsController : ResourceControllerBase
 
                     if (item.AssignedDepartment is null && item.OrgPosition is not null)
                         or.BeResourceOwner(new DepartmentPath(item.OrgPosition.BasePosition.Department).GoToLevel(3), includeDescendants: true);
+
+                    if (!request.AssignedDepartment.HasValue && !request.ProposedPersonAzureUniqueId.HasValue)
```

Comment:
The request creator is not really dependant on any props.. it should be added as a default check..

Reply:
It was meant to handle these lines from the excel sheet. Should the excel sheet be updated or should they be implemented differently?
- Reassign to department
- Propose